main/src/vtr_radar/include/vtr_radar/detector/detector.inl

```diff
@@ -34,8 +34,58 @@ bool sort_asc_by_second(const std::pair<int, float> &a,
   return (a.second < b.second);
 }
 
-}  // namespace
+} // namespace
 
+// // old K-strongest
```

Comment:
Please delete old code. Don't just comment it out

Reply:
The old one is a different implementation of k-strongest involving a mean factor
I keep both as Elliots is a simplified version of the old and that is what I will test online

Reply:
I would still prefer that old code is removed. In the final commit message you can describe the change so that it is searchable.

Reply:
ok